docs/userGuide/contentAuthoring.md

```diff
@@ -113,6 +113,34 @@ In addition, it supports:
 
 * More media blocks, embedding services and additional options can be found in [Markdown-it-block-embed docs](https://github.com/rotorz/markdown-it-block-embed)
 
+### Use MarkBind variables
+
+While authoring content, you can insert variables anywhere in your site. MarkBind treats any text surrounded with double curly braces as a variable, e.g. <code>{<span></span>{variable_name}}</code>. It then replaces variables with pre-defined values when generating the site.

```

Comment:
Combine the first and last sentences into:

"You can insert variables to be replaced by pre-defined values anywhere in your site."